larrecodnn/NuGraph/Tools/StandardLoader_tool.cc

```diff
@@ -0,0 +1,136 @@
+#ifndef STANDARDLOADER_CXX
+#define STANDARDLOADER_CXX
+
+#include "LoaderToolBase.h"
+
+#include "canvas/Persistency/Common/FindManyP.h"
+#include "canvas/Persistency/Common/Ptr.h"
+#include "canvas/Utilities/InputTag.h"
+#include "lardataobj/RecoBase/SpacePoint.h"
+#include <torch/torch.h>
+
+class StandardLoader : public LoaderToolBase {
+
+public:
+  /**
+   *  @brief  Constructor
+   *
+   *  @param  pset
+   */
+  StandardLoader(const fhicl::ParameterSet& pset);
+
+  /**
+   *  @brief  Virtual Destructor
+   */
+  virtual ~StandardLoader() noexcept = default;
+
+  /**
+   *  @brief Interface for configuring the particular algorithm tool
+   *
+   *  @param ParameterSet  The input set of parameters for configuration
+   */
+  void configure(const fhicl::ParameterSet&);
```

Comment:
Suggest removing `configure` entirely and initializing the variables in the initialization list of the constructor.